test/event_test.js

```diff
@@ -1,221 +1,108 @@
-suite('event', function() {
-  var Promise     = require('promise');
-  var launch      = require('../bin/server');
-  var SockJS      = require('sockjs-client-node');
-  var assert      = require('assert');
-  var slugid      = require('slugid');
-  var taskcluster = require('taskcluster-client');
-  var debug       = require('debug')('test:event');
-  var base        = require('taskcluster-base');
+'use strict'
```

Comment:
Ah, we're not even using babel in this project at all. This is fine for now, but we should add babel soon.